ecl/network/v2/load_balancers/results.go

```diff
@@ -0,0 +1,137 @@
+package load_balancers
+
+import (
+	"github.com/nttcom/eclcloud"
+	"github.com/nttcom/eclcloud/pagination"
+)
+
+type commonResult struct {
+	eclcloud.Result
+}
+
+// Extract is a function that accepts a result and extracts a Load Balancer resource.
+func (r commonResult) Extract() (*LoadBalancer, error) {
+	var s struct {
+		LoadBalancer *LoadBalancer `json:"load_balancer"`
+	}
+	err := r.ExtractInto(&s)
+	return s.LoadBalancer, err
+}
+
+// CreateResult represents the result of a create operation. Call its Extract
+// method to interpret it as a Load Balancer.
+type CreateResult struct {
+	commonResult
+}
+
+// GetResult represents the result of a get operation. Call its Extract
+// method to interpret it as a Load Balancer.
+type GetResult struct {
+	commonResult
+}
+
+// UpdateResult represents the result of an update operation. Call its Extract
+// method to interpret it as a Load Balancer.
+type UpdateResult struct {
+	commonResult
+}
+
+// DeleteResult represents the result of a delete operation. Call its
+// ExtractErr method to determine if the request succeeded or failed.
+type DeleteResult struct {
+	eclcloud.ErrResult
+}
+
+// Interface represents a attached interfaces by Load Balancer.
+type Interface struct {
+	ID                  string `json:"id"`
+	IPAddress           string `json:"ip_address"`
+	Name                string `json:"name"`
+	NetworkID           string `json:"network_id"`
+	SlotNumber          int    `json:"slot_number"`
+	Status              string `json:"status"`
+	Type                string `json:"type"`
+	VirtualIPAddress    string `json:"virtual_ip_address"`
```

Comment:
Seems like default value is null though I cannot find official documents.
https://ecl.ntt.com/en/documents/api-references/network/load_balancer/load_balancer.html#id14
```suggestion
	VirtualIPAddress    *string `json:"virtual_ip_address"`
```